tests/src/Kernel/Migrate/OgFieldD7MigrateTest.php

```diff
@@ -44,23 +46,48 @@ class OgFieldD7MigrateTest extends MigrateDrupal7TestBase {
   protected function setUp() {
     parent::setUp();
 
+    $this->fileMigrationSetup();
     $this->loadFixture(__DIR__ . '/../../../fixtures/drupal7.php');
+
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('comment');
+    $this->installEntitySchema('taxonomy_term');
     $this->installConfig(static::$modules);
+
     $this->createNodeCommentCombination('page');
     $this->createNodeCommentCombination('article');
     $this->createNodeCommentCombination('blog');
     $this->createNodeCommentCombination('book');
     $this->createNodeCommentCombination('forum', 'comment_forum');
     $this->createNodeCommentCombination('test_content_type');
+
     Vocabulary::create(['vid' => 'test_vocabulary'])->save();
+
     $this->executeMigrations([
       'd7_field',
       'd7_field_instance',
+      'd7_user_role',
+      'd7_user',
+      'd7_node_type',
+      'd7_node',
       'd7_og_group',
       'd7_og_field_instance',
     ]);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getFileMigrationInfo()
+  {
+    return [
+      'path' => 'public://sites/default/files/cube.jpeg',
+      'size' => '3620',
+      'base_path' => 'public://',
+      'plugin_id' => 'd7_file',
+    ];
+  }
+
   /**
    * Asserts that the og-related fields were migrated properly.
    */
```

Comment:
In testOgAudienceFields we should assert that something with `$formDisplay = EntityFormDisplay::load('node.article.og_audience');` or whatever the id actually is going to be after migration. That and EntityViewDisplay. For both article and forum. Then we know that the additional migrations you added are functional.

Reply:
Sorry for the late reply. I was on vacation :)

I fixed the coding standard issues and added two assertions for the form and view displays.
I added those assertions in OgMembershipTypeD7MigrateTest.php as the migrations affect the form and view displays of the default membership type, not the og_audience field.